### ISSUE TITLE
Deprecate `force_clipboard: true` in favour of `force_mode: clipboard`

### ISSUE THREAD

**Issue by AucaCoyan** [federico-terzi/espanso#1939]
Talking about it in [one of the forums](https://github.com/espanso/espanso/discussions/1690#discussioncomment-9032057) we found out that `force_clipboard` should be deprecated.

We may choose to send a warning as a notification for one option, to give users time to change the option, but because is something so simple to change, I would rather not walk the extra mile for this

**Comment by smeech** [comment]
Should I mention it in the docs, as a deprecated term? Are we likely to remove it from the code at some point?

**Comment by AucaCoyan** [comment]
Nah, it didn't achived a lot of public reception, so probably some users don't even know about it. I don't have a PR to fix it, so in my opinion I would leave as it is and the moment we start working on it we can take action to "deprecate it" officially.
I would also like to know what other team members think about it 😄

**Comment by smeech** [comment]
Thanks. That was my thinking.

**Comment by pyxelr** [comment]
There's a discrepancy in the documentation regarding the `force_mode` option. Specifically, it's mentioned in the [Injection Mechanism section of the Basics documentation](https://espanso.org/docs/matches/basics/#injection-mechanism), but it is missing from the [Options Reference](https://espanso.org/docs/configuration/options/#options-reference).

**Comment by smeech** [comment]
> There's a discrepancy in the documentation regarding the `force_mode` option. Specifically, it's mentioned in the [Injection Mechanism section of the Basics documentation](https://espanso.org/docs/matches/basics/#injection-mechanism), but it is missing from the [Options Reference](https://espanso.org/docs/configuration/options/#options-reference).

Options Reference is about configuration. `force_mode:` just relates to triggers. I don't think it has a place in the configuration  section, but feel free to let me know where you think it should go!

**Comment by pyxelr** [comment]
> There's a discrepancy in the documentation regarding the `force_mode` option. Specifically, it's mentioned in the [Injection Mechanism section of the Basics documentation](https://espanso.org/docs/matches/basics/#injection-mechanism), but it is missing from the [Options Reference](https://espanso.org/docs/configuration/options/#options-reference).
> 
> Options Reference is about configuration. `force_mode:` just relates to triggers. I don't think it has a place in the configuration section, but feel free to let me know where you think it should go!

Ah, got it! So `force_mode` isn’t meant to go in `default.yml`, but rather under each `trigger` or `triggers`, as shown in [this example](https://espanso.org/docs/matches/basics/#keyboard-triggers). Thanks for clarifying, and please disregard my earlier suggestion.
